src/locutil.rs

```diff
@@ -0,0 +1,39 @@
+// TODO import lineBreakG fn from whitespace
+// TODO define types for all fn parameters and structs
+// TODO add getLineInfo fn
+
+// These are used when `options.locations` is on, for the
+// `startLoc` and `endLoc` properties.
+
+mod locutil {
```

Comment:
As discussed, you will need to remove this.

Reply:
yes.